test/e2e

```diff
@@ -44,23 +59,23 @@ get_node_rhel_version() {
 
 # Check if driver-toolkit imagestream is available
 test_imagestream() {
-    oc get imagestream/driver-toolkit -n openshift -o yaml > ${ARTIFACT_DIR}/driver-toolkit-imagestream.yaml
+    oc get imagestream/driver-toolkit -n $TEST_NS -o yaml > ${ARTIFACT_DIR}/driver-toolkit-imagestream.yaml
```

Comment:
`imagestream/driver-toolkit` is an imagestream which is part of the release (the test is only testing its content).
It will be deployed in the `openshift` namespace by the installer.
```suggestion
    oc get imagestream/driver-toolkit -n $openshift -o yaml > ${ARTIFACT_DIR}/driver-toolkit-imagestream.yaml
```